xml_inscriptions/metadata_only/MO.Col.UM.MAA.L.59.18.xml

```diff
@@ -142,7 +162,18 @@
         
         <body>
         <div type="edition">
-            <ab>Transcription goes here</ab></div>
+            <div type="textpart" subtype="face" n="a">
+                <ab>
+                    <lb n="1"/><persName type="imperial"><name type="praenomen" key="Imperator"><expan><abbr>Imp</abbr><ex>erator</ex></expan></name> <name type="cognomen" key="Volusianus">Volusianus</name> <name type="cognomen" key="Augustus"><expan><abbr>Aug</abbr><ex>gustus</ex></expan></name></persName>, 
```

Comment:
Not entirely sure that imperator, which is properly a title, should be considered as a praenomen.